documentation/modules/con-securing-kafka-authentication.adoc

```diff
@@ -96,6 +108,20 @@ use the `networkPolicyPeers` property.
 Use network policies as part of the listener authentication configuration.
 Each listener can have a different `networkPolicyPeers` configuration.
 
-For more information, refer to the xref:con-common-configuration-listener-network-policy-reference[Listener network policies] section and the {K8sNetworkPolicyPeerAPI}.
+For more information, refer to the xref:configuration-listener-network-policy-reference[Listener network policies] section and the {K8sNetworkPolicyPeerAPI}.
 
 NOTE: Your configuration of Kubernetes must support ingress `NetworkPolicies` in order to use network policies in Strimzi.
+
+== Additional listener configuration options
+
+Add further listener configuration using `configuration` properties.
+You can use:
+
+* `bootstrap` and `brokers` properties to specify overrides for connecting to bootstrap and broker services
+* `brokerCertChainAndKey` to xref:proc-installing-certs-per-listener-str[configure a listener to use your own private key and server certificate]
+* `externalTrafficPolicy`, for `nodeport` and `loadbalancer` listeners, to specify whether a service routes external traffic to a local node or cluster-wide endpoints
+* `loadBalancerSourceRanges`, for `loadbalancer` listeners, to specify Classless Inter-Domain Routing (CIDR) ranges to connect to
+* `preferredNodePortAddressType`, for `nodeport` listeners, to specify the address type to use as the node address
+* `useServiceDnsDomain`, for `internal` listeners, to specify whether the Kubernetes service DNS domain is used
+
+For more information, see the xref:type-GenericKafkaListenerConfiguration-reference[GenericKafkaListenerConfiguration schema].
```

Comment:
Does this really belong here? This section is about authentication. Wouldn't it be better to just say something like:

```
For more information about listener configuration and list of all options, see the xref:type-GenericKafkaListenerConfiguration-reference[GenericKafkaListenerConfiguration schema].
```

without listing all the options.

Reply:
It was added for anyone not so familiar with listener config that were other options available. 
But I guess a note with a single link is just as effective :+1: